<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch=""/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open_Image"/>
   <addaction name="action_Open_Diretory"/>
   <addaction name="actionGallery"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="action_Open_Diretory">
   <property name="text">
    <string>Open Diretory</string>
   </property>
  </action>
  <action name="actionGallery">
   <property name="text">
    <string>Gallery</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
